escaper.go

```diff
@@ -0,0 +1,29 @@
+package dbsql
+
+import (
+	"database/sql/driver"
+	"fmt"
+	"strings"
+	"time"
+)
+
+const (
+	KTimeFmt = "2006-01-02T15:04:05.999-07:00"
```

Comment:
What's the `K` prefix for?

Reply:
I used to name constants with leading `k`/`K`. what's the preferred way of naming constants here?

Reply:
Looking at other places where constants were used in the code and general golang guidance, I think this should be `TimeFmt` or even `timeFmt`.